tests/t_ethereum_evm_precompiles.nim

```diff
@@ -34,16 +34,37 @@ const
   TestVectorsDir* =
     currentSourcePath.rsplit(DirSep, 1)[0] / "protocol_ethereum_evm_precompiles"
 
+const
+  TrustedSetupMainnet =
+    currentSourcePath.rsplit(DirSep, 1)[0] /
+    ".." / "constantine" /
+    "commitments_setups" /
+    "trusted_setup_ethereum_kzg4844_reference.dat"
+
 proc loadVectors(TestType: typedesc, filename: string): TestType =
   let content = readFile(TestVectorsDir/filename)
   result = content.fromJson(TestType)
 
-template runPrecompileTests(filename: string, funcname: untyped, outsize: int) =
+proc trusted_setup*(): ptr EthereumKZGContext =
+  ## This is a convenience function for the Ethereum mainnet testing trusted setups.
+  ## It is insecure and will be replaced once the KZG ceremony is done.
```

Comment:
actually the trusted setup being loaded now is the official one from the KZG ceremony
```suggestion
```